javalin/src/test/java/io/javalin/TestResponse.kt

```diff
@@ -196,4 +197,60 @@ class TestResponse {
         assertThat(response.length).isEqualTo(150)
     }
 
+    private fun getLargeSeekableInput(prefixSize: Long, contentSize: Long) = object : InputStream() {
```

Comment:
Could you move this to a separate file?

Reply:
Done. (Not 100% sure about the class name)

Reply:
Thanks! The name isn't too important, I just wanted to keep the test file tidy :)